apps/dashboard/app/views/shared/_bad_cluster_config.html.erb

```diff
@@ -0,0 +1,7 @@
+<%- invalid_clusters.each do |cluster| -%>
+   <%- cluster.errors.each do |error| -%>
+   <div class="alert alert-danger" role="alert">
+      cluster <strong><%= cluster.id.to_s -%></strong> is unusable with error <%= error -%>
```

Comment:
How about this message prefix:

    The cluster config for <strong><%= "cluster.title.to_s (#{cluster.id.to_s})" -%></strong> has a problem:

Reply:
Oh sorry `cluster.title` is not yet defined so that would be omitted.

Reply:
only `id` and `errors` get populated because you can't read the contents of the file. (id is the filename)

Reply:
I could still do 

`The cluster config for <%= cluster.id.to_s -%>.yml has a problem: <%= error -%>`

I could go either way with the `<strong>`  emphasis.  This also assumes that the file extension is `yml` when it can now be `yaml`.

Reply:
So then...

    The cluster config for <strong><%= cluster.id.to_s" -%></strong> has a problem: